src/pages/about.js

```diff
@@ -15,9 +17,36 @@ import ListItemAvatar from '@material-ui/core/ListItemAvatar';
 import Avatar from '@material-ui/core/Avatar';
 
 const useStyles = makeStyles({
-  card: {
+  aboutPage: {
+    display: 'grid',
+    placeItems: 'center',
+    gridTemplateRows: '1fr auto',
+    gridTemplateColumns: '100%'
+  },
+  aboutCards: {
+    maxWidth: '800px',
```

Comment:
Shouldn't we just let the cards here fill the regular max width we have for everything on this site?